frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/_cdrom-modal.scss

```diff
@@ -0,0 +1,32 @@
+.kv-cdrom-modal-padding {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.kv-cdrom-modal-restart-alert {
+  margin-bottom: 10px;
+}
```

Comment:
We can remove this too. In PF we don't use fixed pixel spacing like this. I don't think we need it anyway, but in the future you could use something like `var(--pf-global--spacer--xs)`. I can chat more about Patternfly if that's helpful.